event-protocol/ruby/Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
+gemspec
+
+gem "cucumber", path: "../../../cucumber-ruby"
```

Comment:
This might work on your machine, but it [breaks the build](https://travis-ci.org/cucumber/cucumber/builds/216494566#L285).

I suggest depending on the git repo (with a particular branch) instead.